### PR TITLE
[#28] Docs/users api

### PR DESCRIPTION
**Pull request by JonathanSwallow**

### CODE REVIEW

File: docs/content/usage/users.mdx [hunk 1]
```diff
@@ -15,8 +15,125 @@ The Users API allows you to interact with our users data
 
 ## Usage
 
+### list
```
Comment (Smudge3806) [inline docs/content/usage/users.mdx:18]:
```suggestion
### List
```

Comment (Smudge3806) [reply docs/content/usage/users.mdx:18]: Is List paginated?

**Review by Smudge3806 — CHANGES_REQUESTED** [review]
One minor typo correction and one question

File: docs/content/usage/users.mdx (hunk 1 above)
Comment (JonathanSwallow) [reply docs/content/usage/users.mdx:18]: List is not paginated :+1:

**Review by Smudge3806 — APPROVED** [review]

**Pull request merged**